util/p9/pres

```diff
@@ -0,0 +1,24 @@
+#! /bin/sh
+
+######################################################################################
+# call this script with the name of the result directory you want to copy (i.e. `./res PERF_1555458238`)
```

Comment:
Please make this a usage response that is emitted if the argument is not supplied
```
USAGE='call this script with the name of the result directory you want to copy (i.e. `./res PERF_1555458238`)'
EXPECTED_ARGS=1
if [ $# -ne $EXPECTED_ARGS ]
then
    echo $USAGE
    exit 1
fi
```

Reply:
Great suggestion! Updated.